### PR TITLE
NT Tournaments Wiki page

### PR DESCRIPTION
**Pull request by fajar13k**
Start of NWT 2019 (and NST 2019)

---

### CODE REVIEW

File: wiki/Tournaments/NT/NWT_2019/en.md [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![][flag_AU] [loveleft](https://osu.ppy.sh/users/9240047) |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | ![][flag_DE] [\[Lucky\]](https://osu.ppy.sh/users/1303685) |
+
+---
+
+## Mappools
+
+### Finals
+
+**[Download the mappack here! (123 MB)](https://www.mediafire.com/file/9835f3t5z8xa3a0/Grand+Finals+Pool.zip)**
```
Comment (sr229) [inline wiki/Tournaments/NT/NWT_2019/en.md]: Is this supposed to be map pack instead of mappack?

File: wiki/Tournaments/NT/NWT_2019/en.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+- FreeMod
+  - [DJ Sharpnel - IVALTEK (happy30) \[HappyMiX\]](https://osu.ppy.sh/beatmapsets/50429#osu/154988)
+  - [toby fox & RichaadEB - Megalovania Dual Mix (Fatfan Kolek) \[Insane\]](https://osu.ppy.sh/beatmapsets/763337#osu/1605059)
+  - [Shounen Radio - neu (Philippines) \[Gold\]](https://osu.ppy.sh/beatmapsets/58422#osu/180104)
+- Tiebreaker
+  - **[Yousei Teikoku - Kokou no Sousei (Kalibe) \[Despair\]](https://osu.ppy.sh/beatmapsets/744554#osu/1569836)**
+
+### Semifinals
+
+**[Download the mappack here! (103 MB)](https://www.mediafire.com/file/b1jjruemesunoxe/Semi+Finals+Pool.zip)**
```
Comment (sr229) [inline wiki/Tournaments/NT/NWT_2019/en.md]: Ditto

File: wiki/Tournaments/NT/NWT_2019/en.md [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+- FreeMod
+  - [TAG underground - POSSESSION (miccoliasms) \[EXPERT\]](https://osu.ppy.sh/beatmapsets/40306#osu/127944)
+  - [sak respect for Arata Iiyoshi - Reason of being (CookieBite) \[Cookie M i N's EXTREME\]](https://osu.ppy.sh/beatmapsets/640558#osu/1440742)
+  - [Nizikawa - F.K.S (Necroluttah) \[INFINITE\]](https://osu.ppy.sh/beatmapsets/345519#osu/764510)
+- Tiebreaker
+  - **[Katakiri Rekka - Moe Ochiru Hokori -Counter raid Another D- (\[ Sharuresu \]) \[Final Retribution\]](https://osu.ppy.sh/beatmapsets/511505#osu/1087337)**
+
+### Quarterfinals
+
+**[Download the mappack here! (115 MB)](https://www.mediafire.com/file/8jip2t4vltq7qht/QF+Pool.zip)**
```
Comment (sr229) [inline wiki/Tournaments/NT/NWT_2019/en.md]: Ditto

File: wiki/Tournaments/NT/NWT_2019/en.md (hunk 1 above)
Comment (Joehuu) [reply wiki/Tournaments/NT/NWT_2019/en.md]: Check other tournaments, it's really inconsistent atm.

**Comment by sr229** [comment]
I think we should formalize the spelling for map packs, beatmap sets, etc. else we'll be in a whole lot of trouble consistency-wise.

**Comment by WalterToro** [comment]
@sr229 i agree but that should be something to be pointed as an issue and probably to create a sub article related to tournaments on the ASC

File: wiki/Tournaments/NT/NSST_2019/en.md [hunk 4]
```diff
@@ -0,0 +1,842 @@
...
+| Low | ![][flag_US] [Crihs](https://osu.ppy.sh/users/6995880), ![][flag_CO] [ElMick33](https://osu.ppy.sh/users/5458323), ![][flag_US] [Tuco](https://osu.ppy.sh/users/11706350), ![][flag_ID] [Ulat Bulu](https://osu.ppy.sh/users/3484548), ![][flag_GB] [unrealLyyy](https://osu.ppy.sh/users/9594849), ![][flag_CA] [PayneTrain](https://osu.ppy.sh/users/10275038), ![][flag_NL] [Swerro](https://osu.ppy.sh/users/4507667), ![][flag_CA] [Saltystick](https://osu.ppy.sh/users/2165408) |
+| Unseeded | ![][flag_DE] [Katsumi](https://osu.ppy.sh/users/513834), ![][flag_GB] [4Circ](https://osu.ppy.sh/users/2161823), ![][flag_KR] [siroitora](https://osu.ppy.sh/users/9146185), ![][flag_PL] [lokser](https://osu.ppy.sh/users/4675441), ![][flag_PL] [Kusiu](https://osu.ppy.sh/users/5285044), ![][flag_RU] [Ontoryran](https://osu.ppy.sh/users/5182623), ![][flag_CA] [Xeli](https://osu.ppy.sh/users/4947796), ![][flag_IT] [Condizionatore](https://osu.ppy.sh/users/8810254) |
+
+## Groups
+
+### Tier 1
+
+| Group | Top Seed | High Seed | Low Seed | Unseeded |
+| :-: | :-- | :-- | :-- | :-- | :-- |
```
Comment (WalterToro) [inline wiki/Tournaments/NT/NSST_2019/en.md]:
```suggestion
| :-: | :-- | :-- | :-- | :-- |
```

File: wiki/Tournaments/NT/NSST_2019/en.md [hunk 5]
```diff
@@ -0,0 +1,842 @@
...
+| **E** | ![][flag_US] [im a fancy lad](https://osu.ppy.sh/users/4908650) | ![][flag_US] [Tohru](https://osu.ppy.sh/users/3596296) | ![][flag_DE] [LwL](https://osu.ppy.sh/users/3556856) | ![][flag_SG] [moosepi](https://osu.ppy.sh/users/1868745) |
+| **F** | ![][flag_CA] [Eddie-](https://osu.ppy.sh/users/3898396) | ![][flag_AU] [Dumii](https://osu.ppy.sh/users/3068044) | ![][flag_AR] [Penguo](https://osu.ppy.sh/users/4389490) | ![][flag_IT] [umii](https://osu.ppy.sh/users/2538695) |
+| **G** | ![][flag_US] [Leftoverss](https://osu.ppy.sh/users/4665746) | ![][flag_AR] [-Nanahira-](https://osu.ppy.sh/users/6169195) | ![][flag_AR] [N Y Q A](https://osu.ppy.sh/users/5420232) | ![][flag_US] [Tekkito](https://osu.ppy.sh/users/7075211) |
+| **H** | ![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050) | ![][flag_US] [mniminwoo](https://osu.ppy.sh/users/3929529) | ![][flag_NL] [Lilily](https://osu.ppy.sh/users/6502403) | ![][flag_US] [Will Stetson](https://osu.ppy.sh/users/4909088) |
+
+### Tier 2
+
+| Group | Top Seed | High Seed | Low Seed | Unseeded |
+| :-: | :-- | :-- | :-- | :-- | :-- |
```
Comment (WalterToro) [inline wiki/Tournaments/NT/NSST_2019/en.md]:
```suggestion
| :-: | :-- | :-- | :-- | :-- |
```

**Pull request merged**